openstack_ansibleee/Dockerfile

```diff
@@ -18,10 +18,15 @@ COPY --from=builder /usr/share/ansible /usr/share/ansible
 
 COPY $REMOTE_SOURCE/openstack_ansibleee/settings /runner/env/settings
 RUN chmod 775 /runner/env/settings
-COPY $REMOTE_SOURCE/openstack_ansibleee/edpm_entrypoint.sh /bin/edpm_entrypoint
-RUN sed -i '1d' /bin/entrypoint
-RUN cat /bin/entrypoint >> /bin/edpm_entrypoint
-RUN chmod +x /bin/edpm_entrypoint
+COPY $REMOTE_SOURCE/openstack_ansibleee/edpm_entrypoint.sh /opt/builder/bin/edpm_entrypoint
+RUN sed -i '1d' /opt/builder/bin/entrypoint
+RUN cat /opt/builder/bin/entrypoint >> /opt/builder/bin/edpm_entrypoint
+RUN chmod +x /opt/builder/bin/edpm_entrypoint
 ENV EDPM_SYSTEMROLES='fedora.linux_system_roles'
+RUN chmod ug+rw /etc/passwd
+RUN mkdir -p /runner && chgrp 0 /runner && chmod -R ug+rwx /runner
 WORKDIR /runner
-ENTRYPOINT ["edpm_entrypoint"]
+LABEL ansible-execution-environment=true
+USER 1000
```

Comment:
this ends up adding my local user to the container image. that doesn't seem right.